vud_widget/src/Plugin/VoteUpDownWidgetBase.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Drupal\vud_widget\Plugin;
+
+use Drupal\Component\Plugin\PluginBase;
+
+/**
+ * Base class for Vote Up/Down Widget plugins.
+ */
+abstract class VoteUpDownWidgetBase extends PluginBase implements VoteUpDownWidgetInterface
+{
+
+  /**
+   * VoteUpDownWidgetBase constructor.
+   * @param array $configuration
+   * @param string $plugin_id
+   * @param mixed $plugin_definition
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition) {
```

Comment:
Removed this constructor. I actually saw it in a post but never really got the logic. Seemed irrelevant to me too but kept it anyhow thinking about peer advice